webextensions/api/browserAction.json

```diff
@@ -907,7 +907,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/setTitle",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "8"
```

Comment:
❌ https://github.com/chromium/chromium/commit/1288ba0949c899dce33fc6fcc8e212aee704bc48#diff-38ec526f61a9698ad092b3cfa4ab51adde5dc846633a6f3312314686693ca1b8R348 landed in [4.0.223.0](https://github.com/chromium/chromium/releases/tag/4.0.223.0), which is the release tag (4.0.223).

```suggestion
                "version_added": "4"
```